### PR TITLE
Added Operator Lookup Link in Learning Section 

### PR DESCRIPTION
**Pull request by Aditi-1400**
Added operator lookup link in external learning section as per issue #1223.
Screenshot from 2021-03-26 00-24-51
I've added the same in index.ja.md, but since I couldn't see any external learning section in index page of other languages, I did not make any changes in them.

### CODE REVIEW

File: site/learn/tutorials/index.md [hunk 1]
```diff
@@ -117,3 +117,7 @@ Both in french:
   on using PPX, the syntax extensions API.
 * [Detecting use cases for GADTs in OCaml](http://mads-hartmann.com/ocaml/2015/01/05/gadt-ocaml.html),
   on using generalized algebraic data types in writing interpreters.
+* [Ocaml Operator Lookup](https://www.craigfe.io/operator-lookup/),
```
Comment (patricoferris) [inline site/learn/tutorials/index.md]: To be consistent with the rest of the site, it should be `OCaml` and not `Ocaml`.

File: site/learn/tutorials/index.md [hunk 2]
```diff
@@ -117,3 +117,7 @@ Both in french:
...
+  for getting an explanation on behaviour of a operator or syntax element, along with usage examples.
```
Comment (patricoferris) [inline site/learn/tutorials/index.md]: I think this might read better as "on the behaviour" and it should be "an operator"

**Review by patricoferris — CHANGES_REQUESTED** [review]
Thanks @Aditi-1400 :)) A few suggestions then it should be good to go.

**Comment by avsm** [comment]
Thanks for the contribution! Now merged to the live site.

**Pull request merged**